pkg/ovsdb/transact.go

```diff
@@ -228,7 +228,10 @@ func NewTransaction(cli *clientv3.Client, request *libovsdb.Transact) *Transacti
 	txn.cache = Cache{}
 	txn.schemas = libovsdb.Schemas{}
 	txn.request = *request
-	txn.response.Result = make([]libovsdb.OperationResult, len(request.Operations))
+	txn.response.Result = make([]*libovsdb.OperationResult, len(request.Operations))
+	for i, _ := range txn.response.Result {
+		txn.response.Result[i] = libovsdb.NewOperationResult()
```

Comment:
This `NewOperationResult` call creates an instance of OperationResult which marshals  to JSON string as 
{"count":0,"error":"","details":"","uuid":["named-uuid",""],"rows":null}.
In order to be inline with the spec, I suggest just to create an empty OperationResult {}, and during the operations, just initialize the relevant field.